atest/interpreter.py

```diff
@@ -48,6 +49,23 @@ def _get_name_and_version(self):
         version = re.match(r'\d+\.\d+\.\d+', version).group()
         return name, version
 
+    def _get_java_version(self):
+        if not self.is_jython:
+            return (-1, -1)
+        try:
+            # platform.java_ver() returns Java version in a format:
+            # ('9.0.7.1', 'Azul Systems, Inc.', ('OpenJDK 64-Bit Server VM', '9.0.7.1+1', 'Azul Systems, Inc.'), ('Windows 10', '10.0', 'amd64'))
+            # ('11.0.6', 'Ubuntu', ('OpenJDK 64-Bit Server VM', '11.0.6+10-post-Ubuntu-1ubuntu118.04.1', 'Ubuntu'), ('Linux', '4.4.0-18362-Microsoft', 'amd64'))
+            # ('1.8.0_121', 'Oracle Corporation', ('Java HotSpot(TM) 64-Bit Server VM', '25.121-b13', 'Oracle Corporation'), ('Windows 10', '10.0', 'amd64'))
+            script = 'import platform; print(platform.java_ver()[0])'
+            output = subprocess.check_output(self.interpreter + ['-c', script],
+                                             stderr=subprocess.STDOUT,
+                                             encoding='UTF-8')
+        except (subprocess.CalledProcessError, FileNotFoundError):
+            raise ValueError('Invalid interpreter: %s' % self.path)
+        major, minor = output.strip().split('.', 2)[:2]
+        return (int(major), int(minor))
```

Comment:
I believe parentheses around `(int(major), int(minor))` are redundant

Reply:
Yeah. I always forget about that. Fixed.